packages/gatsby/src/commands/develop.js

```diff
@@ -428,7 +428,14 @@ module.exports = async (program: any) => {
   }
 
   function printInstructions(appName, urls, useYarn) {
-    console.log()
+    report._setStage({
+      stage: `BootstrapFinished`,
+      context: {
+        url: urls.localUrlForBrowser,
+        appName,
+      },
+    })
+    console.log(`\x1B[2J\x1B[0f`)
```

Comment:
what's this? :p maybe add a comment

Reply:
clears the screen :-) yeah, good idea — added comment.